widget/diagramwidget/anchoredtext.go

```diff
@@ -42,7 +43,7 @@ func NewAnchoredText(text string) *AnchoredText {
 	at.displayedTextBinding.AddListener(at)
 	at.textEntry.Wrapping = fyne.TextWrapOff
 	// TODO After upgrade to fyne 2.4.0, uncomment the following line and add container as an imported package
```

Comment:
I think this TODO should have been removed

Reply:
Done